sesam.py

```diff
@@ -207,7 +207,7 @@ def wait_for_all_pipes_to_deploy(self, timeout=30*60):
             time.sleep(5)
 
     def is_user_pipe(self, pipe):
-        return self.get_pipe_origin(pipe) not in ["system", "replica", "aggregator-storage-node"]
+        return self.get_pipe_origin(pipe) not in ["system", "search", "replica", "aggregator-storage-node"]
```

Comment:
An alternative, and more future proof way, is to just do: `return self.get_pipe_origin(pipe) == "user"`

Reply:
The origin metadata is primarily mean for internal use by Sesam.

Reply:
Will implement. 👍